index.html

```diff
@@ -44,5 +44,160 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
+            <div class="project-txt">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <img src="./images/project3.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Tonic</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <img src="./images/project4.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button> 
+            </div>
+        </div>
+    </section>
+    <!-- About section -->
+    <section class="about">
+        <div class="about-content">
+            <h1 class="about-title">About <br> Myself</h1>
```

Comment:
- [ ] Similar to the comment about heading above, kindly change this to `<h2>` to` <h6>`. I would recommend `<h2>` since this heading is second level heading next to the `<h1>` heading in the headline.